backend/blueprints/spa_api/service_layers/player/player_profile_stats.py

```diff
@@ -11,11 +12,12 @@
 
 
 class PlayerProfileStats:
-    def __init__(self, favourite_car: str, car_percentage: float):
+    def __init__(self, favourite_car: str, car_percentage: float, players_in_common: list):
```

Comment:
I think you should make a PlayerInCommonStat class to mirror the frontend. So this will take `players_in_common: List[PlayerInCommonStat]` and then do something like `self.playersInCommon = player_in_common.__dict__ for player_in_common in players_in_common`